go.mod

```diff
@@ -4,16 +4,16 @@ go 1.15
 
 require (
 	github.com/docker/distribution v2.7.1+incompatible
-	github.com/go-logr/logr v0.3.0
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.2
+	github.com/go-logr/logr v0.4.0
+	github.com/onsi/ginkgo v1.15.0
+	github.com/onsi/gomega v1.10.5
 	github.com/opencontainers/go-digest v1.0.0 // indirect
-	k8s.io/api v0.20.2
-	k8s.io/apiextensions-apiserver v0.20.1
-	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v0.20.2
-	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
-	sigs.k8s.io/controller-runtime v0.8.0
+	k8s.io/api v0.21.0
+	k8s.io/apiextensions-apiserver v0.21.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/client-go v0.21.0
+	k8s.io/utils v0.0.0-20210305010621-2afb4311ab10
+	sigs.k8s.io/controller-runtime v0.9.0-alpha.1.0.20210409160505-5031262be76f
```

Comment:
@sunjayBhatia v0.9.x is for testing only. I think v0.8.3 would be a better choice, unless I'm missing something that's needed in 0.9.0.

Reply:
This was done because of https://github.com/kubernetes-sigs/controller-runtime/issues/1473

Reply:
Compilation fails with 0.8.3

Reply:
That's a good enough reason to go with v0.9.x.